v1/examples/inputs/dependent-field/case-3-pipeline.yaml

```diff
@@ -1,34 +1,25 @@
 # A runtime input field depends on other runtime input value field and fixed value field
 
-pipeline:
-  name: test
-  identifier: test
-  description: description
-  projectIdentifier: hotstar
-  orgIdentifier: a_org
-  tags: {}
+version: 1
+kind: pipeline
+spec:
   stages:
-    - stage:
-        name: test
-        identifier: test
-        description:
-        type: Custom
-        spec:
-          execution:
-            steps:
-              - step:
-                  type: JiraCreate
-                  name: JiraCreate_1
-                  identifier: JiraCreate_1
-                  spec:
-                    connectorRef: dummy-connectorRef
-                    projectKey: <+inputs.input-1>
-                    issueType: <+inputs.input-2>
-                    fields: []
-                  timeout: 10m
-        tags: {}
+    - name: test
+      type: custom
+      spec:
+        steps:
+          - name: JiraCreate_1
+            type: JiraCreate
+            id: JiraCreate_1
+            spec:
+              connectorRef: dummy-connectorRef
+              projectKey: <+inputs.input-1>
```

Comment:
this is invalid it needs tp be either <+inputs["input-1"]> or  <+inputs.get("input-1")>

Reply:
is that because we have `-` in our input name i.e. `input-1`?
is it was `input_1`, would this `<+inputs.input_1>` work?

Reply:
yes `<+inputs.input-1>` the problem is how do u interpret this can be a vart called input minus 1

Reply:
okay, got it, updated the examples